src/components/members-list/index.js

```diff
@@ -1,20 +1,40 @@
+/* eslint-disable consistent-return */
+/* eslint-disable array-callback-return */
```

Comment:
If possible can we find some ways to implement code without disabling linting rules

Reply:
+1